css/src/atomics/colors.scss

```diff
@@ -0,0 +1,225 @@
+	$hover: nth($color-set, $color-index-hover);
+	$active: nth($color-set, $color-index-active);
+	$invert: nth($color-set, $color-index-invert);
+
+	.color-#{$name} {
+		color: $base !important;
+	}
+
+	a.color-#{$name} {
+		&:hover,
+		&.focus-visible {
+			color: $hover !important;
+		}
+
+		&:active {
+			color: $active !important;
+		}
+	}
+
+	.color-#{$name}-invert {
+		color: $invert !important;
+	}
+
+	.background-color-#{$name} {
+		outline-color: $invert;
+		background-color: $base !important;
+	}
+
+	.background-color-#{$name}-tablet {
+		@include tablet {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-desktop {
+		@include desktop {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-widescreen {
+		@include widescreen {
+			outline-color: $invert;
+			background-color: $base !important;
+		}
+	}
+
+	.background-color-#{$name}-invert {
+		outline-color: $base;
+		background-color: $invert !important;
+	}
+
+	.background-color-#{$name}-light {
+		outline-color: $dark;
+		background-color: $background !important;
+	}
+
+	.background-color-#{$name}-dark {
+		outline-color: $background;
+		background-color: $dark !important;
+	}
+}
+
+// Text color
+
+.color-text {
+	color: $text !important;
+}
+
+.color-text-subtle {
+	color: $text-subtle !important;
+}
+
+.color-text-overlay-invert {
+	color: $overlay-invert;
+}
+
+.color-current-color {
+	color: currentColor !important;
+}
+
+.fill-current-color {
+	fill: currentColor !important;
+}
+
+.outline-color-text {
+	outline-color: $text !important;
+}
+
+.stroke-current-color {
+	stroke: currentColor !important;
+}
+
+// Background color
+
+.background-color-main {
+	outline-color: $text;
+	background-color: $body-background !important;
+}
+
+.background-color-main-tablet {
+	@include tablet {
+		outline-color: $text;
+		background-color: $body-background !important;
+	}
+}
+
+.background-color-main-desktop {
+	@include desktop {
+		outline-color: $text;
+		background-color: $body-background !important;
+	}
+}
+
+.background-color-main-widescreen {
+	@include widescreen {
+		outline-color: $text;
+		background-color: $body-background !important;
+	}
+}
+
+.background-color-main-medium {
+	outline-color: $text;
+	background-color: $body-background-medium !important;
+}
+
+.background-color-main-medium-tablet {
+	@include tablet {
+		outline-color: $text;
+		background-color: $body-background-medium !important;
+	}
+}
+
+.background-color-main-medium-desktop {
+	@include desktop {
+		outline-color: $text;
+		background-color: $body-background-medium !important;
+	}
+}
+
+.background-color-main-medium-widescreen {
+	@include widescreen {
+		outline-color: $text;
+		background-color: $body-background-medium !important;
+	}
+}
+
+.background-color-main-dark {
+	outline-color: $text;
+	background-color: $body-background-dark !important;
+}
+
+.background-color-main-dark-tablet {
```

Comment:
do we want the css classes to match the variable name?

Reply:
`.background-color-body-dark-tablet`?

Reply:
Updated `main` to `body`.